resources/lang/vi/auth.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Authentication Language Lines
+|--------------------------------------------------------------------------
+|
+| The following language lines are used during authentication for various
+| messages that we need to display to the user. You are free to modify
+| these language lines according to your application's requirements.
+|
+*/
+
+return [
+    'failed'   => 'Thông tin tài khoản không tìm thấy trong hệ thống.',
```

Comment:
```suggestion
    'failed' => 'Thông tin tài khoản không tìm thấy trong hệ thống.',
```
sửa lại để 1 dấu cách thôi nha.